specs/abbr.spec.md

```diff
@@ -0,0 +1,77 @@
+# Abbr (abbreviation)
+
+## Overview
+
+From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr), the `<abbr>` HTML element represents an abbreviation or acronym. Much of this specification borrows heavily from the details provided by MDN. The primary *need* for this element is that it relies on the native browser tooltip which cannot be styled. Design systems and web authors looking for consistency across experiences may find a `<fast-attr>` element beneficial as it would enable comparable support with customizable UI.
```

Comment:
```suggestion
From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr), the `<abbr>` HTML element represents an abbreviation or acronym. Much of this specification borrows heavily from the details provided by MDN. The primary *need* for this element is that it relies on the native browser tooltip which cannot be styled. Design systems and web authors looking for consistency across experiences may find a `<fast-abbr>` element beneficial as it would enable comparable support with customizable UI.
```